src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -1175,6 +1262,79 @@ RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1,
 #endif // backend
 }
 
+/******************** glitch ********************/
+
+RppStatus rppt_glitch_gpu(RppPtr_t srcPtr,
+                            RpptDescPtr srcDescPtr,
+                            RppPtr_t dstPtr,
+                            RpptDescPtr dstDescPtr,
+                            Rpp32u *xOffsetR,
+                            Rpp32u *yOffsetR,
+                            Rpp32u *xOffsetG,
+                            Rpp32u *yOffsetG,
+                            Rpp32u *xOffsetB,
+                            Rpp32u *yOffsetB,
+                            RpptROIPtr roiTensorPtrSrc,
+                            RpptRoiType roiType,
+                            rppHandle_t rppHandle)
+{
+#ifdef HIP_COMPILE
+Rpp32u paramIndex = 0;
+copy_param_uint(xOffsetR, rpp::deref(rppHandle), paramIndex++);
+copy_param_uint(yOffsetR, rpp::deref(rppHandle), paramIndex++);
+copy_param_uint(xOffsetG, rpp::deref(rppHandle), paramIndex++);
```

Comment:
Lot of copies here, @sampath1117 I think we have to consider taking one 2D tensor called rgbOffsets[batchSize]. And access as rgbOffsets[n].rOffset.x, rgbOffsets[n].rOffset.y
A single argument is passed rgbOffsets, and it could be in pinned memory removing these copies.